blog-post.html

```diff
@@ -1,12 +1,51 @@
+

```

Comment:
This page seems to need a formatter ran. When I ran one on my end it didn't clean anything up which generally means that you have a miss matched open/close tag.